Form/form.py

```diff
@@ -61,6 +61,8 @@
 
 # The key of the data item in a source to define it as a form source.
 DEFINITION_KEY = _('Form')
+# the following should be all the translations of "Form" in our po files
+INTL_FORM = {"Form", "Formular", "Formulaire", "Obrazac", "Форма"}
```

Comment:
In case a new langauge is created, can a comment be added to template.pot to remind the translator to also get this list updated?